developerportal/apps/topics/tests/test_topics.py

```diff
@@ -44,6 +44,14 @@ class TopicsTests(PatchedWagtailPageTests):
 
     fixtures = ["common.json"]
 
+    def setUp(self):
+        # purge any Topic titles we unintentionally cache
+        cache.clear()
+
+    def tearDown(self):
+        # purge any Topic titles we unintentionally cache
+        cache.clear()
+
```

Comment:
This is a fix to a test regression that shows up when re-running tests that now get Topics